src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -157,6 +158,57 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
     />
   ));
 
+  if (isMobile) {
+    const SelectedIcon = PlatformIcons[selected || 'Builder'];
+    return (
+      <Box sx={{ position: 'relative' }}>
+        <SpeedDial
+          ariaLabel='SpeedDial basic example'
+          sx={{
+            position: 'absolute',
+            '& button': {
+              width: '40px',
+              height: '40px',
+            },
```

Comment:
You can remove this part if you pass the `size="small"` prop to the FabProps prop. It renders the Floating Action button (the component that powers the main speed dial button as a "dense" button.